mig-agent/config.go

```diff
@@ -167,9 +183,30 @@ type globals struct {
 
 	// Add the private client key below.
 	agentKey []byte
+
+	// let you target specific agents
+	tags map[string]string
+}
+
+func globalTagsMap() map[string]string {
+	tags := make(map[string]string)
+	inspectTags := reflect.TypeOf(&TAGS).Elem()
+
+	for tag, val := range structs.Map(TAGS) {
+		field, ok := inspectTags.FieldByName(tag)
+
+		if ok {
+			_, tag = stringPair(string(field.Tag))
+			tag = strings.Replace(tag, `"`, "", -1)
+		}
+		tags[tag] = fmt.Sprintf("%v", val)
+	}
+	return tags
```

Comment:
@jvehent does this sound reasonable to you? I'm not sure what the original intent around TAGS being an arbitrary struct was. Do you see any issue if we just convert this so TAGS is assumed to be map[string]string, since generally that is how we are using it across the board anyway?